### PR TITLE
Add install instructions for gentoo

### PR DESCRIPTION
**Pull request by mroethke**

### CODE REVIEW

File: source/usage.rst [hunk 1]
```diff
@@ -64,6 +64,24 @@ First, setup Homebrew [#homebrew]_ and afterwards install the RTRlib package:
 
 .. [#homebrew]  Homebrew -- http://brew.sh
 
+
+Gentoo
+------
+
+The kind guys from the Frrouting project maintain a gentoo overlay [#overlay]_ that contains an ebuild for rtrlib.
+First you have to setup layman [#layman]_, then you can install rtrlib with the following commands:
```
Comment (smlng) [inline source/usage.rst]: for consistency (and brevity) skip `you` like above in the macOS + Homebrew instructions.

File: source/usage.rst [hunk 2]
```diff
@@ -64,6 +64,24 @@ First, setup Homebrew [#homebrew]_ and afterwards install the RTRlib package:
 
 .. [#homebrew]  Homebrew -- http://brew.sh
 
+
+Gentoo
+------
+
+The kind guys from the Frrouting project maintain a gentoo overlay [#overlay]_ that contains an ebuild for rtrlib.
```
Comment (smlng) [inline source/usage.rst]: typos (?) please fix/replace `/Frrouting/FRRouting/` and `/rtrlib/the RTRlib/`

**Review by smlng — CHANGES_REQUESTED** [review]
minor wording nits

**Comment by mroethke** [comment]
ping @smlng

**Comment by smlng** [comment]
please squash

**Review by smlng — APPROVED** [review]

**Comment by mroethke** [comment]
Done

**Pull request merged**